examples/bri-3/src/bri/zeroKnowledgeProof/components/blockchain/blockchain.service.ts

```diff
@@ -0,0 +1,26 @@
+import { Injectable, OnModuleInit } from '@nestjs/common';
+import { ZeroKnowledgeProofVerificationInput } from '../../models/zeroKnowledgeProofVerificationInput';
+
+@Injectable()
+export class BlockchainService implements OnModuleInit {
+  async onModuleInit() {
```

Comment:
it  does not need onModuleInit as the connection with the blockchain will be created only when needed